basic-counter/app.js

```diff
@@ -0,0 +1,23 @@
+class App extends React.Component {
```

Comment:
All these files :point_down: shouldn't be included in this PR

Double check what you are commiting by using `git status` before you commit.

Reply:
ops